plugins/nv-filters/data/locale/en-US.ini

```diff
@@ -6,10 +6,14 @@ Nvafx.Method.Denoiser="NVIDIA Noise Removal"
 Nvafx.Method.Dereverb="NVIDIA Room Echo Removal"
 Nvafx.Method.DenoiserPlusDereverb="NVIDIA Noise Removal + Room Echo Removal"
 Nvafx.OutdatedSDK="WARNING: Please upgrade both NVIDIA Video & Audio SDK. Your current version of Audio SDK is outdated."
+Nvafx.VAD="Voice Optimized"
+Nvafx.VAD.Hint="Optimizations for audio data which contains a voice"
```

Comment:
This might just be me, but that tooltip seems redundant because it just says the same thing in more words. 

Does it only optimise spoken text or also vocal performance? Does it isolate a single voice or is able to optimise for multiple speakers?

Reply:
removed the tooltip; it's designed for speech. With singing, it does an odd job (can get the voice but tries to remove the instruments). If there are multiple speakers who speak in alternance, it will work well for all of them. When several people are speaking at the same time, they all pass through (tested on a YT video of a debate with two people speaking over each other)